docs/platforms/android/profiling/index.mdx

```diff
@@ -11,139 +11,250 @@ sidebar_order: 5000
   Profiling uses the Android runtime's `tracer` under the hood to sample threads. There are known issues that this `tracer` can cause crashes in certain circumstances. See this <PlatformLink to="/profiling/troubleshooting#i-see-elevated-number-of-crashes-in-the-android-runtime-when-profiling-is-activated">troubleshooting</PlatformLink> entry for more information.
 </Alert>
 
-## Enable Tracing
+## Installation
 
-Profiling depends on Sentry’s Tracing product being enabled beforehand. To enable tracing in the SDK:
+Android profiling is available starting in SDK version `8.7.0`.
+The legacy profiler is available on SDK versions `6.16.0` and higher through <PlatformLink to="/configuration/options/#legacy-profiling-options">its options</PlatformLink>.
 
-In `AndroidManifest.xml`:
+## Enabling Profiling
 
-```xml
-<application>
-  <meta-data android:name="io.sentry.dsn" android:value="___PUBLIC_DSN___" />
-  <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
-</application>
-```
+Profiling supports two modes - `manual` and `trace`. The two modes are mutually exclusive, and cannot be used at the same time.
 
-Check out the <PlatformLink to="/tracing/">tracing setup documentation</PlatformLink> for more detailed information on how to configure sampling. Setting the sample rate to 1.0 means all transactions will be captured.
+In `manual` mode, the profiling data collection can be managed via calls to `Sentry.profiler.startProfiler` and `Sentry.profiler.stopProfiler`. You are entirely in the in control of when the profiler runs.
 
-By default, some transactions will be created automatically for common operations like loading a view controller/activity and app startup.
+In `trace` mode, the profiler manages its own start and stop calls, which are based on spans: the profiler continues to run while there is at least one active sampled span, and stops when there are no active sampled spans.
 
-## Enable Profiling
+Sentry SDK supports an additional `profileSessionSampleRate` that will enable or disable profiling for the entire <PlatformLink to="/configuration/releases/#sessions">session</PlatformLink>. This sampling decision is evaluated only once per session.
 
-<Alert>
+<Alert title="Note">
 
-Android profiling is available starting in SDK version `6.16.0` and is supported on API level 22 and up.
-App start profiling is available starting in SDK version `7.3.0`.
+Continuous profiling has implications for your org's billing structure. This feature is only available for subscription plans that enrolled after June 5, 2024.
```

Comment:
We should remove this because UI Profiling will be available on AM2

Reply:
@Zylphrex fyi